vscode/package.json

```diff
@@ -409,20 +409,59 @@
         ],
         "firstLine": "^#!\\s*/.*(?:ruby|rbx|rake)\\b",
         "extensions": [
+          ".builder",
+          ".eye",
+          ".fcgi",
           ".gemspec",
+          ".god",
           ".irbrc",
           ".jbuilder",
+          ".mspec",
+          ".pluginspec",
+          ".podspec",
+          ".prawn",
           ".pryrc",
+          ".rabl",
           ".rake",
           ".rb",
+          ".rbi",
+          ".rbuild",
+          ".rbw",
+          ".rbx",
           ".ru",
-          ".ruby"
+          ".ruby",
+          ".spec",
+          ".thor",
+          ".watchr"
         ],
         "filenames": [
+          ".irbrc",
+          ".pryrc",
+          ".simplecov",
+          "Appraisals",
+          "Berksfile",
+          "Brewfile",
+          "Buildfile",
+          "Capfile",
+          "Dangerfile",
+          "Deliverfile",
+          "Fastfile",
+          "Gemfile",
```

Comment:
As It is not explained in [documentation](https://code.visualstudio.com/api/references/contribution-points#contributes.languages) I did a few more tests to be sure, it's not case-sensitive! If I delete `Jarfile`, it's not detected as a ruby file, but `JaRfILE` works. I've deleted all duplicate occurrences, keeping only uppercase, because it's more "in the way" IMO. Feel free to challenge!

As a side note, many of them still are detected as ruby files even with a complete omission from this list (`Gemfile`, `Rakefile`, `Thorfile`, etc), probably a VS Code behavior coming from their [languages detection](https://github.com/Microsoft/vscode-languagedetection).

Reply:
Thank you for double-checking! I agree, we should use the most common names and the other defaults do come from VS Code's built-in Ruby language configuration.